synapseclient/client.py

```diff
@@ -198,27 +198,33 @@ class Synapse(object):
     """
     Constructs a Python client object for the Synapse repository service
 
-    :param repoEndpoint:          Location of Synapse repository
-    :param authEndpoint:          Location of authentication service
-    :param fileHandleEndpoint:    Location of file service
-    :param portalEndpoint:        Location of the website
-    :param serviceTimeoutSeconds: Wait time before timeout (currently unused)
-    :param debug:                 Print debugging messages if True
-    :param skip_checks:           Skip version and endpoint checks
-    :param configPath:            Path to config File with setting for Synapse
-                                  defaults to ~/.synapseConfig
-    :param requests_session:      a custom requests.Session object that this Synapse instance will use
-                                  when making http requests
-
-    Typically, no parameters are needed::
+    Attributes:
```

Comment:
This is what it'll look like to convert the existing docstrings into google style. This style gives us the biggest flexibility and interoperability with mkdocstrings.

Reply:
Agreed, I think we can have an epic around converting everything to google style docs, and those can be good-first-tickets for the community to contribute to.

It'll also be good tickets for all of us to contribute.